### DISCUSSION TITLE
How can I use Ant Design's default colors in my React components?

### DISCUSSION THREAD

**Discussion by shaimaHamila** [ant-design/ant-design#47648] [Q&A]
I'm new to Ant Design, and I'm looking for guidance on how to use Ant Design's default colors (like volcano-6, lime-8, etc.) in my React components. Can someone provide a simple example or explanation on how to achieve this? Thank you!

**Comment by zombieJ** [comment]
https://ant.design/docs/react/customize-theme#consume-design-token
